frontends/bmd/src/components/printed_ballot.tsx

```diff
@@ -135,7 +132,12 @@ function CandidateContestResult({
   election,
   precinctId,
   vote = [],
-}: CandidateContestResultInterface): JSX.Element {
+}: {
+  contest: CandidateContest;
+  election: Election;
+  precinctId: PrecinctId;
+  vote: CandidateVote;
+}): JSX.Element {
```

Comment:
Would it make sense to inline these interfaces in this file but name them `CandidateContestResultProps`, etc? That's the usual convention I've seen for component prop typing (as opposed to inlining the object type)